### PR TITLE
AdminModerators page with elements and methods

### PR DESCRIPTION
**Pull request by AhnessP**
Covered Am1 - Am6 & Ac1-Ac7 except for Ac3 (date picker)

### CODE REVIEW

**Review by eugenek7 — APPROVED** [review]

**Review by Slim1011 — APPROVED** [review]

File: EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+        public void ClickBirthDateInputField()
+        {
+            _birthDateInputField.Click();
+        }
+        
+        public void SendKeysToNameInputField(string name)
+        {
+            _nameInputField.SendKeys(name);
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs]: please remove useless line

File: EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        }
+        
+        public void SendKeysToEmailInputName(string email)
+        {
+            _emailInputField.SendKeys(email);
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs]: please remove useless line

File: EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        }
+        
+        public void SendKeysToInputQuantityReorderField(string phone_number)
+        {
+            _phoneNumberInputField.SendKeys(phone_number);
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs]: please remove useless line

File: EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        }
+        
+        public void SendKeysToPasswordInputField(string password)
+        {
+            _passwordInputField.SendKeys(password);
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs]: please remove useless line

File: EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        }
+        
+        public void SendKeysToRepeatPasswordInputField(string repeat_password)
+        {
+            _repeatPasswordInputField.SendKeys(repeat_password);
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/AdminModeratorsPage.cs]: please remove useless line

**Review by dmatsev — APPROVED** [review]
Approve, but please resolve the comments before merging

**Pull request merged**